src/isar/config/config.py

```diff
@@ -1,17 +1,24 @@
+import logging
 from configparser import ConfigParser
 from os import getenv
 from pathlib import Path
 
 from dotenv import load_dotenv
+
 from isar.config.configuration_error import ConfigurationError
 
 
 class Config(object):
     def __init__(self):
         load_dotenv()
         env = getenv("ENVIRONMENT")
+
         if not env:
-            raise ConfigurationError(f"Environment not set")
+            env = "local"
+            logging.warning(
+                "ENVIRONMENT has not been set.\n Defaulting to local environment."
+            )
```

Comment:
Not sure this needs to be a warning. Maybe logging.info is sufficient? Could be a valid use case to just rely on the local environment.